### PR TITLE
Fixes for gitlab pages. Fixes #1158

### LINKED ISSUE #1158: Enabling Pages creates bad config
Adding these EV:
* GITLAB_PAGES_ENABLED: true
* GITLAB_PAGES_DOMAIN: mysub.myfirm.org
* GITLAB_PAGES_PORT: 8090

```
28.3.2017 16:39:502017-03-28 14:39:50,678 INFO exited: nginx (exit status 1; not expected)
28.3.2017 16:39:532017-03-28 14:39:53,342 INFO spawned: 'nginx' with pid 3221
28.3.2017 16:39:532017-03-28 14:39:53,376 INFO exited: nginx (exit status 1; not expected)
28.3.2017 16:39:562017-03-28 14:39:56,385 INFO spawned: 'nginx' with pid 3223
28.3.2017 16:39:562017-03-28 14:39:56,422 INFO exited: nginx (exit status 1; not expected)
28.3.2017 16:39:572017-03-28 14:39:57,424 INFO gave up: nginx entered FATAL state, too many start retries too quickly
```

### PR DESCRIPTION
**Pull request by phpb-com**

### CODE REVIEW

**Comment by f3l1x** [comment]
Good job. Thanks.

**Comment by diegobernardes** [comment]
nice, really needed this feature! now just need to wait the pr merge :+1:

File: assets/runtime/functions [hunk 1]
```diff
@@ -965,15 +965,22 @@ nginx_configure_gitlab_hsts() {
...
-  fi
-}
+ nginx_configure_gitlab_ipv6() {
+   if [[ ! -f /proc/net/if_inet6 ]]; then
+     # disable ipv6 support in nginx for gitlab
+     sed -i \
+       -e "/listen \[::\]:80/d" \
+       -e "/listen \[::\]:443/d" \
+       ${GITLAB_NGINX_CONFIG}
+     # disable ipv6 support in nginx for pages
```
Comment (solidnerd) [inline assets/runtime/functions]: I guess we could remove this and directly from `assets/nginx/gitlab_pages`

**Comment by solidnerd** [comment]
LGTM ! 

This fixes are needed. I didn't test the integration when registry were enabled during the implementation. Thanks for that and the feed. I will merge this directly on master. It will be available on the latest tag first. Feedback is welcome.

File: assets/runtime/functions (hunk 1 above)
Comment (phpb-com) [inline assets/runtime/functions]: You are right, my bad. Thanks.

**Pull request merged**